test/unit/attribute/OwnerExecutable.test.ts

```diff
@@ -0,0 +1,72 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers'
+import { expect } from 'chai'
+import HRE from 'hardhat'
+
+import { MockOwnerExecutable, MockOwnerExecutable__factory } from '../../../types/generated'
+
+const { ethers } = HRE
+
+describe('OwnerExecutable', () => {
+  let owner: SignerWithAddress
+  let user: SignerWithAddress
+  let unrelated: SignerWithAddress
```

Comment:
Linter picked up that this isn't used anywhere; can it be removed?

Reply:
Fixed in [ef268b3](https://github.com/equilibria-xyz/root/pull/111/commits/ef268b38bf63e7a6c16d1be4fd3f688b2dd812bd)